Userland/Libraries/LibVideo/VP9/ProbabilityTables.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2021, Hunter Salyer <thefalsehonesty@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Constants.h"
+#include <AK/Types.h>
+#include <AK/Vector.h>
+
+namespace Video::VP9 {
+
+typedef u8 pareto_table_t[128][8];
```

Comment:
Is there an advantage to that `using` syntax? I'm not really too familiar with it! And yeah we can probably just drop the `_t` suffix if that works?

Reply:
No particular advantage other than making it a little more readable.
`Array<T, size>` is nice because I personally always think "wait is that 128 rows by 8 columns, or 8 rows by 128 columns?".

if we do drop the `_t`, it'll have to be in CamelCase to match the coding styles:
> Use CamelCase (Capitalize the first letter, including all letters in an acronym) in a class, struct, or namespace name.